finalitygadget/finalitygadget.go

```diff
@@ -230,7 +230,7 @@ func (fg *FinalityGadget) QueryBlockRangeBabylonFinalized(
 	}
 	// handle case where no block is finalized
 	if finalizedBlockHeight == nil {
-		return nil, fmt.Errorf("no block was finalized in the given range")
+		return nil, nil
```

Comment:
this is a logic check. could you provide some context?

likele related to "This PR creates the Dockerfile for the finality gadget server and fixes two small issues."?

Let's explain what these two issues are, or reviewers won't know the context and it will easily be forgotten after a week :)